css/style.css

```diff
@@ -0,0 +1,194 @@
+body {
+    margin-top: 100px;
+    background-color: grey;
+    margin-left: 0;
+}
+
+#header-img {
+    width: 218px;
+    height: 25px;
+    margin-top: 25px;
+    margin-right: 10px;
+}
+
+
+.header-ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+
+.header-ul li {
+    float: right;
+    height: 60px;
+    margin-top: 10px;
+    margin-left: 100px;
+    width: 150px;
+    
+}
+
+.header-ul li a {
+   display: block;
+    margin-top: 15px;
+    height: 60px;
+    font-size: 20px;
+    text-decoration: none;
+}
+
+#header {
+    display: flex;
+    float: right;
+    position: fixed;
+    top: 0;
+    text-align: center;
+    background-color: rgb(176, 224, 224);
+    padding: 0px 20px 0px 100px;
+    justify-content: space-around;
+    width: 100%;
+    box-shadow: 5px 5px 15px 1px #000;
+    
+}
+
+#nav-bar {
+    
+    display: flex;
+    padding: auto;
+    
+    list-style: inside;
+    
+}
+
+.container {
+    margin-left: 25%;
+    margin-right: 25%;
+}
+
+
+@media (max-width: 800px) {
+    header {
+      flex-wrap: wrap;
+    }
+  }
+
+  #box {
+    background-color: lightyellow;
+    box-shadow: 5px 5px 10px 1px #000;
+
+  }
+
+  #video{
+   
+    height: 300px;
+    width: 95%;
+    margin: 10px ;
+
+  }
+
+  #pricing {
+      display:flex;
+      width: 95%;
+      margin-left: 20%;
+      margin-right: 20%;
+
+  }
+
+  .price-structure {
+      padding: 20px;
+      background-color: mediumaquamarine;
+  }
+
+  #box-1, #box-2, #box-3 {
+      border: 5px gray solid;
+  }
+.price-name {
+    background-color: honeydew;
+    width: 100%;
+    text-align: center;
+    margin-top: 10px;
+    padding-top: 15px;
+    
+    display: block;
+}
+
+.button{
+    background-color: yellow;
+    margin: 0 30%;
+
+}
+
+#submit{
+    background-color: tomato;
+    margin-left: 20px;
+    margin-bottom: 20px;
+}
+
+#email {
+    margin-left: 20%;
+}
+
+#form {
+    
+    margin: 0 25%;
+}
+
+.email-box {
+    background-color: rgb(81, 81, 192);
+    box-shadow: 5px 5px 10px 1px #000;
+}
+
+.features-names {
+    margin-left: 25%;
+    margin-right: 25%;
+    
+    text-align: center;
+}
+
+.features-specs {
+    text-align: center;
+    margin-right: 25%;
+    margin-left: 25%;
+}
+
+.end{
+    
+    background-color: wheat;
+    display: flex;
+    margin-top: 30px;
+    padding: 20px;
+    
+    ;
+    
+}
+
+.end ul {
+    list-style-type: none;
+    display: flex ;
+    justify-content: flex-end;
+       background-color: wheat;
+    margin-left:40% ;
+       
+       
+}
+
+
+.end ul li {
+    
+    margin-left: 10px;
+    
+    
+        
+}
```

Comment:
You have a lot of spaces and indentations that aren't necessary for all the CSS file, like this part of the code. You only need 2 spaces of indentation.

Reply:
It would be super helpful if you always give an explanation for your requirements. One of the many objectives code reviews have is to help the person being reviewed, and what better way to accomplish this than to give them a clear reason on why we are suggesting the changes 🧐 Kindly explain in your comments the reason for your requirements 😁

Also, please be sure to tell your requirements/suggestions in a way that it sounds as suggestions instead of orders 😉 The idea is to make it politely and friendly 😬